serverless.yaml

```diff
@@ -22,8 +22,8 @@ custom:
     copyFiles: # Copy any additional files to the generated package
       - from: 'bulkExport/glueScripts/export-script.py'
         to: './bulkExport/glueScripts/export-script.py'
-      - from: 'compiledImplementationGuides/*'
-        to: './'
+      - from: 'compiledImplementationGuides'
+        to: './compiledImplementationGuides'

```

Comment:
This change was made because Windows does not like the `*` wildcard. Error shown below
```
ERROR in unable to locate 'C:\Users\Administrator\Documents\fhir-works-on-aws-deployment\compiledImplementationGuides\*'
 glo
```